posix.mak

```diff
@@ -39,6 +39,10 @@ install: all
 	mkdir -p $(INSTALL_DIR)/man
 	cp -r docs/man/* $(INSTALL_DIR)/man/
 
+check-clean-git:
```

Comment:
Major targets should have documentation explaining what they do. Otherwise, the user is left to puzzle out the commands executed, which (like all code) isn't always obvious.